### PR TITLE
Resolve conflicts with `master`

### PR DESCRIPTION
**Pull request by Sneagan**
### Type of Change

- [ ] Product feature
- [ ] Bug fix
- [ ] Performance improvement
- [ ] Refactor
- [ ] Other

### Tested Environments

- [ ] Development
- [ ] Staging
- [ ] Production

### Before Requesting Review

- [ ] Does your code build cleanly without any errors or warnings?
- [ ] Have you used auto closing keywords?
- [ ] Have you added tests for new functionality?
- [ ] Have validated query efficiency for new database queries?
- [ ] Have documented new functionality in README or in comments?
- [ ] Have you squashed all intermediate commits?
- [ ] Is there a clear title that explains what the PR does?
- [ ] Have you used intuitive function, variable and other naming?
- [ ] Have you requested security and/or privacy review if needed
- [ ] Have you performed a self review of this PR?

### CODE REVIEW

File: services/ratios/mapping.go [hunk 1]
```diff
@@ -10,129 +10,124 @@ import (
...
-		"xor":    "sora",
-		"yld":    "yield",
-		"zap":    "zap",
+	// symbolToUniqueIDMapping - mapping of symbols to unique CoinGecko IDs.
+	//
+	// This is used to handle symbols that are not unique across different chains.
+	// For example, "btc" is used by both Bitcoin and BlackrockTradingCurrency.
+	//
+	// This mapping should only be used when querying by CoinGecko ID is not possible.
+	symbolToUniqueIDMapping = map[string]string{
```
Comment (kdenhartog) [inline services/ratios/mapping.go:19]: How is this being generated? It looks like we're changing this list here.

File: services/skus/controllers.go [hunk 2]
```diff
@@ -3,33 +3,42 @@ package skus
 import (
```
Comment (kdenhartog) [inline services/skus/controllers.go:3]: I'm not sure what's going on with these or how this refactor is going to affect the nitro-payments branch

File: services/wallet/instrumented_datastore.go [hunk 3]
```diff
@@ -265,7 +266,7 @@ func (_d DatastoreWithPrometheus) LinkWallet(ctx context.Context, id string, pro
 
 		datastoreDurationSummaryVec.WithLabelValues(_d.instanceName, "LinkWallet", result).Observe(time.Since(_since).Seconds())
 	}()
-	return _d.base.LinkWallet(ctx, id, providerID, providerLinkingID, depositProvider, country)
+	return _d.base.LinkWallet(ctx, id, providerID, providerLinkingID, depositProvider)
```
Comment (kdenhartog) [inline services/wallet/instrumented_datastore.go:269]: Are we storing this country in some other way now?

File: services/wallet/keystore_test.go [hunk 4]
```diff
@@ -1,530 +0,0 @@
-//go:build integration
-
-package wallet_test
```
Comment (kdenhartog) [inline services/wallet/keystore_test.go:3]: What's the reason for removing these?

**Review by kdenhartog — APPROVED** [review]
I didn't look through the skus changes, but this seems fine for us to merge in to the nitro payments branch. It's a bit hard to identify which of these changes are things to resolve conflicts versus refactors to improve the nitro payments branch (if any of that is the case). 

Once it's been merged in I'm going to take a look at how the branch has changed since my last review and do a re-review of the changes and clean up of the issues we still need to handle.

**Comment by pavelbrm** [comment]
A few PRs for Ntiro have been closed recently. Should this one follow the same path? @Sneagan

**Comment by tackley** [comment]
Is this still relevant?  Or can we close or mark as draft please to remove from our needs-review lists?

**Comment by pavelbrm** [comment]
@tackley I believe this is not relevant anymore, but will leave the decision to @Sneagan.

**Comment by pavelbrm** [comment]
Do we still need this?

**Comment by kdenhartog** [comment]
I don't think so - at this point it seems all payments dev work has been migrated to a separate repo so I believe this is fine to close

**Pull request closed without merging**